pkg/controller/migmigration/backup.go

```diff
@@ -48,6 +48,27 @@ func (t *Task) ensureInitialBackup() (*velero.Backup, error) {
 	return newBackup, nil
 }
 
+// Combine MigPlan excluded resources with velero excluded resources
+func (t *Task) appendMigPlanExcludes(veleroExcludedResources []string) []string {
```

Comment:
I think this can be simplified using set logic.  The Go pattern I have seen for this is using a `map[string]bool` as a `set` (the value can be anything).
1. Add all of the resources in the velero list.
2. Add all of the resources in the plan list.
3. Return the map keys.